src/main/java/com/team4/testingsystem/services/TestsService.java

```diff
@@ -0,0 +1,90 @@
+package com.team4.testingsystem.services;
+
+import com.team4.testingsystem.entities.Test;
+import com.team4.testingsystem.repositories.TestsRepository;
+import com.team4.testingsystem.repositories.UsersRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.NoSuchElementException;
+
+@Service
+public class TestsService {
+
+    private TestsRepository testsRepository;
+
+    private UsersRepository usersRepository;
+
+
+    @Autowired
+    public TestsService(TestsRepository testsRepository, UsersRepository usersRepository) {
+        this.testsRepository = testsRepository;
+        this.usersRepository = usersRepository;
+    }
+
+
+    public Iterable<Test> getAll() {
+        return testsRepository.findAll();
+    }
+
+    public Test getById(long id) {
+            if (!testsRepository.existsById(id)) {
+                throw new NoSuchElementException("Test not found");
+            }
+        return testsRepository.findById(id).get();
+    }
+
+    public long create(long userId) {
```

Comment:
Create method that accepts long seams confusing. Can we somehow rename it to show the logic?